src/resources/boards/board.service.js

```diff
@@ -0,0 +1,64 @@
+const { v4: uuidv4 } = require('uuid');
+const path = require("path");
+const { writeDataToFile } = require('../../utils/fileService');
+const { badRequest, notFound } = require('../../messages/errors');
+
+const dataWay = path.resolve("./data/boards.json")
+// eslint-disable-next-line import/no-dynamic-require
+const boardData = require(dataWay)
```

Comment:
![image](https://user-images.githubusercontent.com/74081058/169669258-62b0367b-30f2-4009-9f5d-85d687baa94e.png)
one tricky solution 
`const boardData = require('${dataWay}')`
not sure but you can try